### PR TITLE
docs: Edits to remove jargon

### PR DESCRIPTION
**Pull request by TeriGlover**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [X] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: aio/content/guide/i18n.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ For more information about `package.json` and polyfill packages, see [Workspace
 
 </div>
 
-If `@angular/localize` is not installed, the Angular CLI may generate an error when you try to build a localized version of your app.
+If `@angular/localize` is not installed, the Angular CLI might generate an error when you try to build a localized version of your app.
```
Comment (petebacondarwin) [inline aio/content/guide/i18n.md:84]: what is the motivation behinding switching from `may` to `might`?

File: aio/content/guide/testing-pipes.md [hunk 2]
```diff
@@ -4,9 +4,9 @@ You can test [pipes](guide/pipes) without the Angular testing utilities.
 
 <div class="alert is-helpful">
 
-  For a hands-on experience you can <live-example name="testing" stackblitz="specs" noDownload>run tests and explore the test code</live-example> in your browser as your read this guide.
+  For a hands-on experience, <live-example name="testing" stackblitz="specs" noDownload>run tests and explore the test code</live-example> in your browser as your read this guide.
```
Comment (petebacondarwin) [inline aio/content/guide/testing-pipes.md:7]:
```suggestion
  For a hands-on experience, <live-example name="testing" stackblitz="specs" noDownload>run tests and explore the test code</live-example> in your browser as you read this guide.
```

File: aio/content/guide/testing-utility-apis.md [hunk 3]
```diff
@@ -336,7 +336,7 @@ Here are the most important static methods, in order of likely utility.
       The testing shims (`karma-test-shim`, `browser-test-shim`) call it for you
       so there is rarely a reason for you to call it yourself.
 
-      You may call this method _exactly once_. If you must change
+      You can call this method _exactly once_. If you must change
       this default in the middle of your test run, call `resetTestEnvironment` first.
```
Comment (petebacondarwin) [inline aio/content/guide/testing-utility-apis.md:340]:
```suggestion
      Call this method _exactly once_. To change
      this default in the middle of a test run, call `resetTestEnvironment` first.
```

**Review by petebacondarwin — CHANGES_REQUESTED** [review]

File: aio/content/guide/i18n.md (hunk 1 above)
Comment (aikidave) [reply aio/content/guide/i18n.md:84]: The simplest reason: to comply with the Google Developer Style Guide: https://developers.google.com/style/word-list?hl=en#might. 

In addition, "may" and "might" can be localized differently; by using "might" we ensure more consistency across documentation sets.

**Review by aikithoughts — APPROVED** [review]
Looks good. Please address Pete's additional changes and this PR should be good to go.

Reviewed-for: global-docs-approvers

**Comment by aikidave** [comment]
@JoostK can you add a comment, `@googlebot I consent` to this PR? Thank you!

**Comment by JoostK** [comment]
@aikidave  it looks like my commit got in here by accident. Can this be rebased to drop my commit?

**Review by petebacondarwin — APPROVED** [review]
LGTM - but did you mean to revert the changes to the links?
E.g.  from

```
[explore the translated example app][AioGuideI18nAppPreTranslation]
```

to

```
[explore the translated example app](#app-pre-translation)
```

If you do want to revert these then you should also remove them from the foot of the file.

**Pull request closed without merging**